js/packages/teams-ai/src/AI.ts

```diff
@@ -108,6 +115,11 @@ export interface ConfiguredAIOptions<TState extends TurnState> {
      * If true, the AI system will allow the planner to loop.
      */
     allow_looping: boolean;
+
+    /**
+     * If true, the AI system will enable the feedback loop in Teams that allows a user to give thumbs up or down to a response.
+     */
+    enableFeedbackLoop: boolean;
```

Comment:
im noticing a mix of camel case and snake case here, we should try to stick to one or the other if we can